src/app/vault/add-edit.component.html

```diff
@@ -71,17 +72,18 @@ <h2 class="modal-title" id="cipherAddEditTitle">{{title}}</h2>
                                 <input id="loginPassword" class="form-control text-monospace"
                                     type="{{showPassword ? 'text' : 'password'}}" name="Login.Password"
                                     [(ngModel)]="cipher.login.password" appInputVerbatim autocomplete="new-password"
-                                    [disabled]="cipher.isDeleted">
+                                    [disabled]="cipher.isDeleted || !cipher.viewPassword">
```

Comment:
Assuming this will change with reversing the logic to carry forward the `disablePassword` theme.